examples/uki/Containerfile.arch

```diff
@@ -4,8 +4,9 @@ COPY cfsctl /usr/bin
 RUN <<EOF
     set -eux
     touch /etc/machine-id
+    mkdir -p boot/EFI/Linux
     pacman -Syu --noconfirm
-    pacman -Sy --noconfirm skopeo composefs strace dosfstools openssh
+    pacman -Sy --noconfirm skopeo composefs strace dosfstools openssh linux
```

Comment:
Ah right.  For the modules, of course.

I guess you should remove the `linux` from the installation in the next stage...

Reply:
You mean the phase where we install `systemd-ukify`?

Reply:
Yes.  Installing the kernel there is now redundant since it's already installed